roles/core/ssh_remote_keys/vars/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-ssh_slave_role_version: 1.1.0
+ssh_slave_role_version: 1.1.1
```

Comment:
Should this role still have this variable names? Perhaps it should be `ssh_remote_keys_version`?

Reply:
+1